src/main/java/reposense/util/FileUtil.java

```diff
@@ -38,6 +41,71 @@
     private static final String GITHUB_API_DATE_FORMAT = "yyyy-MM-dd";
     private static final ByteBuffer buffer = ByteBuffer.allocate(1 << 11); // 2KB
 
+    /**
+     * Returns a list of relevant repo folders to be zipped up later
+     * @param sourcePath contains the source directory where the repo folders are located at
+     * @param configs contains the list of configuration information for a single repository
+     * @return a set of relevant repo folders (to be zipped later)
+     */
+    public static HashSet<Path> getReportFolders(Path sourcePath, List<RepoConfiguration> configs) {
+        HashSet<Path> relevantFolders = new HashSet<>();
+        for (RepoConfiguration repoConfiguration : configs) {
+            relevantFolders.add(Paths.get(sourcePath + File.separator + repoConfiguration.getDisplayName())
+                    .toAbsolutePath());
+        }
+        return relevantFolders;
+    }
+
+    /**
+     * Zips all the relative folders and relevant files of {@code fileTypes} contained in
+     * {@code sourceAndOutputPath} directory into the same folder.
+     * @param pathsToZip contains the folders and files to be zipped.
+     * @param sourceAndOutputPath contains the directory where the source folders and files are located at and
+     *                           where to be zipped to.
```

Comment:
inconsistent wrapping, in fact if you can rename the variables and describe the method more accurately using `@code` in the paragraph, you probably wouldn't need to use `@param` tag